connectors/katalog/utils/utils.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+// GenerateRandomString returns a securely generated random string.
+// It will return an error if the system's secure random
+// number generator fails to function correctly, in which
+// case the caller should not continue.
+// Source: https://gist.github.com/dopey/c69559607800d2f2f90b1b1ed4e550fb
+func GenerateRandomString(n int) (string, error) {
+	const letters = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
you can put this const out of the function
should it be lover case only?
There are faster solutions see https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-go

Reply:
Since the linter suggests using the crypto/rand package, I am using the function provided in https://gist.github.com/dopey/c69559607800d2f2f90b1b1ed4e550fb. I have moved the constant out of the function. Please review. Thanks.